adafruit_atecc/adafruit_atecc.py

```diff
@@ -385,11 +409,13 @@ def sha_start(self):
         self.idle()
         return status
 
-    def sha_update(self, message):
-        """Appends bytes to the message. Can be repeatedly called.
+    def sha_update(self, message: Any) -> bytearray:
```

Comment:
`message` is noted as `bytes` in the docstring just below here. Are there cases where it would be something of a different type?